pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/persistent/PersistentDispatcherMultipleConsumers.java

```diff
@@ -483,6 +484,9 @@ public Consumer getNextConsumer() {
         } while (true);
 
         // not found unblocked consumer
+        if (log.isDebugEnabled()) {
```

Comment:
can 481 - 484 be converted to 

} while(++resultingAvailableConsumerIndex < consumerList.size());